Update _fusion-missing-features.md (#8128)
fishtown-analytics/docs.getdbt.com: website/snippets/_fusion-missing-features.md

| [Active and auto exposures](/docs/cloud-integrations/downstream-exposures) | You rely on auto exposures to pull downstream assets (like Tableau dashboards) into dbt lineage, or on active exposures to proactively refresh downstream assets (like Tableau extracts) during scheduled jobs. <br /><br /> Fusion doesn't support active and auto exposures yet. | [dbt-fusion#704](https://github.com/dbt-labs/dbt-fusion/issues/704)
| [`{{ graph }}`](/reference/dbt-jinja-functions/graph) - `raw_sql` attribute (e.g. specific models in [dbt_project_evaluator](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/)) | You access the `raw_sql` / `raw_code` attribute of the `{{ graph }}` context variable, which Fusion stubs with an empty value at runtime. If you access this attribute, your code will not fail, but it will return different results. This is used in three quality checks within the [`dbt_project_evaluator` package](https://hub.getdbt.com/dbt-labs/dbt_project_evaluator/latest/). We intend to find a more-performant mechanism for Fusion to provide this information in the future. | TK |

> | Externally orchestrated jobs | You use a third-party orchestrator (like Astronomer-Cosmos) that depends on a dbt manifest produced by <Constant name="core"/>. Many of these integrations don't yet support manifests generated by <Constant name="fusion"/>. However, <Constant name="fusion"/> does support external orchestrators that integrate through the <Constant name="dbt_platform"/> run job API. | TK |

Reviewer: what does "TK" mean?

Author: I was just coming here to ask Matt! I followed the same format that the table already had here: https://docs.getdbt.com/docs/fusion/supported-features

The last entry has TK so I just mimicked that since there's no GitHub issue. 

@matthewshaver what does tk mean? A google search says it means 'to come' in editorial publishing

> | Externally orchestrated jobs | You use an external third-party orchestrator dbt integration (like Astronomer-cosmos) that relies on a dbt manifest produced by <Constant name="core"/>. Many of these integrations still need to build support for <Constant name="fusion"/>-generated manifests. However, <Constant name="fusion"/> does support external orchestrators that integrate through the <Constant name="dbt_platform"/> run job API. | TK |

Author: ```suggestion
| Externally orchestrated jobs | You use a third-party orchestrator (like Astronomer-Cosmos) that depends on a dbt manifest produced by <Constant name="core"/>. Many of these integrations don't yet support manifests generated by <Constant name="fusion"/>. However, <Constant name="fusion"/> does support external orchestrators that integrate through the <Constant name="dbt_platform"/> run job API. | TK |
```
